src/dags/deelmobiliteit.py

```diff
@@ -0,0 +1,268 @@
+import operator
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.postgres_operator import PostgresOperator
+from airflow.operators.python_operator import PythonOperator
+from airflow.operators.dummy_operator import DummyOperator
+from common.db import DatabaseEngine
+from environs import Env
+from more_ds.network.url import URL
+from provenance_rename_operator import ProvenanceRenameOperator
+from pgcomparator_cdc_operator import PgComparatorCDCOperator
+from sqlalchemy_create_object_operator import SqlAlchemyCreateObjectOperator
+from typing import Dict
+
+from common import (
+    default_args,
+    slack_webhook_token,
+    DATAPUNT_ENVIRONMENT,
+    MessageOperator,
+)
+
+from postgres_check_operator import (
+    PostgresMultiCheckOperator,
+    COUNT_CHECK,
+    GEO_CHECK,
+)
+
+from sql.deelmobiliteit import SQL_SET_GEOM, SQL_DROP_TMP_TABLE
+from importscripts.import_deelmobiliteit import import_scooter_data, import_auto_data
+
+
+dag_id: str = "deelmobiliteit"
+description: str = (
+    "locaties en contextuele informatie over deelvoertuigen zoals auto’s, fietsen en scooters."
+)
+variables: Dict = Variable.get(dag_id, deserialize_json=True)
+env: Env = Env()
+
+endpoint_scooters: Dict = variables["scooters"]["data_endpoints"]
+fleyx_base_url: str = URL(env("AIRFLOW_CONN_FLEYX_BASE_URL"))
+fleyx_api_key: str = env("AIRFLOW_CONN_FLEYX_API_KEY")
+ridecheck_base_url: str = URL(env("AIRFLOW_CONN_RIDECHECK_BASE_URL"))
+ridecheck_token_url: str = URL(env("AIRFLOW_CONN_RIDECHECK_TOKEN_URL"))
+ridecheck_token_client_id: str = env("AIRFLOW_CONN_RIDECHECK_CLIENT_ID")
+ridecheck_token_client_secret: str = env("AIRFLOW_CONN_RIDECHECK_CLIENT_SECRET")
+
+endpoint_autos: Dict = variables["autos"]["data_endpoints"]
+mywheels_base_url: str = URL(env("AIRFLOW_CONN_MYWHEELS_BASE_URL"))
+mywheels_api_key: str = env("AIRFLOW_CONN_MYWHEELS_API_KEY")
+
+db_conn: object = DatabaseEngine()
+total_checks: list = []
+count_checks: list = []
+geo_checks: list = []
+check_name: dict = {}
+
+
+def quote(instr: str) -> str:
```

Comment:
Deze komt in veel dags terug. Is een kandidaat om in het `common` package te zetten.

Reply:
Check